_events/sr2019/southampton-kickstart.md

```diff
@@ -12,13 +12,25 @@ be announced and robotics kits will be handed out to the teams.
 If you are unable to attend Kickstart, please [let us know][teams-contact] so we
 can arrange another way to get you your kit.
 
+We'll have desktops available for students to use, however feel free to bring personal laptops if you prefer.
+
 ## Directions
 
-* How to [get to campus][soton-campus-directions]
+The Southampton kickstart is being held at the University of Southampton at:
+
+EEE Building (32),  
+University of Southampton,  
+University Road,  
+Southampton,  
+SO17 1BJ  
+
+For Public transport or driving instructions, see the University of Southampton's information on [getting to our campuses][soton-campus-directions] (scroll down to Highfield campus).
```

Comment:
ah, just spotted that "Public" here should probably be lower case.
```suggestion
For public transport or driving instructions, see the University of Southampton's information on [getting to our campuses][soton-campus-directions] (scroll down to Highfield campus).
```

Reply:
I really like this 'suggested changes' feature :+1: